bmwoolard/pull-request-demo
# test my new feature please

## to test

- open index.html file and press the increment button
- verify that count updates by 1 and UI is updated as well
